uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/contact-actions-menu.js

```diff
@@ -0,0 +1,37 @@
+var Promise = require('es6-promise').Promise;
+var page = require('./page');
+
+module.exports = (function () {
+  var contactActionsMenuButtonSelector = '#crm-contact-actions-link';
+  var deleteUserAccountButtonSelector = '[data-delete-user-url]';
+
+  return page.extend({
+
+    /**
+     * Click the "Actions" button on contact page
+     */
+    openContactActionMenu: function () {
+      var casper = this.casper;
+
+      return new Promise(function (resolve) {
```

Comment:
This promise seems redundant, as you are not using `resolve()` anywhere. Also I think this promise is not required at all, because `casper.click` uses `await`internally.

Reply:
Refactored!